tests/framework/PostProcessors/MCSimporterPostProcessor/test_MCSimporter_noExpand.xml

```diff
@@ -1,11 +1,11 @@
 <Simulation verbosity="debug">
   <TestInfo>
-    <name>framework/PostProcessors/MCSimporterPostProcessor.MC_importer_Saphire</name>
+    <name>framework/PostProcessors/MCSimporterPostProcessor.MC_importer_noExpand</name>
```

Comment:
MC_importer_noExpand should be changed to __MCSimporterNoExpand__, the name used in __tests__  file.

Reply:
Fixed!

Reply:
It seems to me the change is not consistent with the __tests__ file. __MCSimporterNoExpand__?